pkg/exporter/sumologicexporter/sender.go

```diff
@@ -418,11 +419,6 @@ func isEmptyAttributeValue(att pdata.AttributeValue) bool {
 // to configured LogFormat and as the result of execution
 // returns array of records which has not been sent correctly and error
 func (s *sender) sendLogs(ctx context.Context, flds fields) ([]logPair, error) {
-	// Follow different execution path for OTLP format
-	if s.config.LogFormat == OTLPLogFormat {
-		return s.sendOTLPLogs(ctx, flds)
```

Comment:
Not sure if it's plausible but maybe return an error just in case?

Reply:
I.e. 
```
	if s.config.LogFormat == OTLPLogFormat {
		return nil, fmt.Errorf("OTLPLogFormat was attempted to be sent as non-OTLP data")
```

Or some sort of assertion that it is not OTLPLogFormat

Reply:
Added